audio-worklet/src/noisegate-script-processor.js

```diff
@@ -80,7 +80,7 @@ class NoiseGate {
     // The last weight (between 0 and 1) assigned, where 1 means the gate
     // is open and 0 means it is closed and the sample in the output buffer is
     // muted. When attacking, the weight will linearly decrease from 1 to 0, and
-    // when releasing the weight linearly increase from 0 to 1. 
+    // when releasing the weight linearly increase from 0 to 1.
```

Comment:
No - since attacking in electronic music refers to the onset of an event, and here the event is the muting of the signal or a 0 weight.